README.md

```diff
@@ -77,7 +81,7 @@ alt="platformatic"
    Or manually create the file as ESM:
 
    ```js
-   import { neostandard } from 'neostandard'
+   import neostandard from 'neostandard'
```

Comment:
I'm fine with supporting default exports but should we nudge towards named exports?

Reply:
For ESM uses I agree, the default if any would be best to have as simply the default config then, as one could then do a one line export of it